basis-universal-sys/vendor/transcoding_wrapper.cpp

```diff
@@ -157,7 +157,32 @@ extern "C" {
     // basisu_lowlevel_uastc_transcoder
     //
 
-    // Not implemented
+    struct LowLevelUastcTranscoder {
+        basist::basisu_lowlevel_uastc_transcoder *pTranscoder;
+    };
+
+    LowLevelUastcTranscoder *low_level_uastc_transcoder_new() {
```

Comment:
Extremely minor nit, I'd prefer the new and delete functions to be adjacent so that if one of them is modified, it will be more likely the person modifying it will see and remember to update the corresponding function

Reply:
Done.